Fix UG related errors from PE-D (#260)
AY2425S1-CS2103T-F15-2/tp: docs/UserGuide.md

6. Refer to the [Features](#features) below for details of each command.

--------------------------------------------------------------------------------------------------------------------

## Summary of a `Person`

This table will explain the fields that a `Person` in EduContacts possesses and its respective constraints. Each person

> in EduContacts is assumed to be Singaporean. All fields should not be blank upon the initialisation of a Student, except for `Module`.

Reviewer: ```suggestion
in EduContacts is assumed to be Singaporean. You should not leave any fields blank when adding a person to EduContacts, except for `Module`.
```

I think we can make this line more user-centric as such. I also think we should change student to person since this is a summary of a `Person`. Using student in this context could confuse users.

Field      | Details
-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
**StudentID**   | The StudentID that belongs to the `Person`. The input for this field can only contain digits and should be exactly 8 digits long. <br><br> This field also serves as the unique identifier for a `Person`.

> **Name**   | The name that belongs to the `Person`. The input for this field can only contain alphanumeric characters.
> **Course** | The course that the `Person` studies. The input for this field can only contain alphabetical characters.

Reviewer: ```suggestion
**Name**   | The name that belongs to the `Person`. The input for this field can only contain alphanumeric characters and whitespaces.
**Course** | The course that the `Person` studies. The input for this field can only contain alphabetical characters and whitespaces.
```
I think we should specify that these two fields allow whitespaces to differentiate from the other fields that do not allow whitespaces e.g. module
